compiler-plugin/src/test/kotlin/arrow/meta/quotes/scope/templates/IsExpressionTest.kt

```diff
@@ -50,19 +76,7 @@ class IsExpressionTest  {
     assertThis(CompilerTest(
       config = { listOf(addMetaPlugins(IsExpressionPlugin())) },
       code = { source },
-      assert = { compiles }//quoteOutputMatches(source) }
+      assert = { compiles }// }
```

Comment:
```suggestion
      assert = { quoteOutputMatches(source) }
```

The quote output must be validated for all the tests.

Reply:
thanks! is done.

Reply:
Thanks to you!! :+1: